.github/workflows/bikes.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         kubectl create namespace $GITHUB_HEAD_REF
         kubectl label namespace $GITHUB_HEAD_REF azds.io/parent-space=${{ secrets.MASTER_SPACE }} 
         kubectl label namespace $GITHUB_HEAD_REF azds.io/space=true
+        kubectl label namespace $GITHUB_HEAD_REF azds.io/pull-request-flow=true
```

Comment:
I think it should just be pull-request=true (no flow)

Reply:
Updated the label.

Reply:
Thinking more: should we label this as azds.io/pull-request-space?